scoap3/articles/admin.py

```diff
@@ -231,6 +226,21 @@ def get_identifiers(self, obj):
         )
 
 
+def make_compliant(article_id):
+    try:
+        article = Article.objects.get(id=article_id)
```

Comment:
`obj.report.all()` or `obj.report.first()` can do the trick, since the article can be passed form the queryset